src/ActionRepeater.UI/Views/ActionListView.xaml

```diff
@@ -26,7 +26,7 @@
                    Margin="67,0,45,1"
                    VerticalAlignment="Bottom"
                    HorizontalAlignment="Right"
-                   Text="show key auto-repeat actions"/>
+                   Text="Show character repeat actions"/>
```

Comment:
```suggestion
                   Text="Show key auto-repeat actions"/>
```
here the old text is more accurate, "key auto-repeat" is what windows calls they auto generated key-downs when you hold a key